examples/foc/foc_adc.h

```diff
@@ -66,7 +66,7 @@
 #ifdef CONFIG_EXAMPLES_FOC_SETPOINT_CONST
 #  define SETPOINT_ADC_SCALE   (1)
 #  if CONFIG_EXAMPLES_FOC_SETPOINT_CONST_VALUE == 0
-#    error
+/* #    error */
```

Comment:
can we merge the patch in  nuttx first?

Reply:
I think that if we merge first the patch in nuttx then CI will be broken until this PR merge.

related PR https://github.com/apache/incubator-nuttx/pull/4744

Reply:
Both patch can't pass the CI, should we merge them together?

Reply:
It looks like it's the only possibility.
In that case I remove the last commit, due the fact that CI fails for another reason and please merge this PR together with https://github.com/apache/incubator-nuttx/pull/4744

Reply:
Ok, let's merge both together.